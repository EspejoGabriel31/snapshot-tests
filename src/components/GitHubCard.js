import React, {useEffect, useState} from 'react'
import Card from 'react-bootstrap/Card'
import PFP from './githubPFP.png'

export default function GitHubCard(){

    return(
        <div>
            <Card style={{width:'18rem'}}>
                <Card.Img variant="top" src={PFP}/>
                <Card.Body>
                    <Card.Title>
                        Gabriel Espejo
                    </Card.Title>
                    <Card.Text>
                        Software engineering degree haver, pretend storyteller, vtuber fan
                    </Card.Text>
                </Card.Body>
            </Card>
        </div>
    )
}