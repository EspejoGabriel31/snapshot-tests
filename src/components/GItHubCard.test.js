import { render, screen } from '@testing-library/react';
import GitHubCard from './GitHubCard';
import renderer from 'react-test-renderer'

test('create snapshot', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})