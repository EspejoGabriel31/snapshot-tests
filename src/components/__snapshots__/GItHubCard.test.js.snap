// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot 1`] = `
<div>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="githubPFP.png"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Gabriel Espejo
      </div>
      <p
        className="card-text"
      >
        Software engineering degree haver, pretend storyteller, vtuber fan
      </p>
    </div>
  </div>
</div>
`;
